wallet/src/actors/app/handlers/sign_data.rs

```diff
@@ -0,0 +1,28 @@
+use actix::prelude::*;
+use serde::{Deserialize, Serialize};
+
+use crate::actors::app;
+use crate::{model, types};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct SignDataRequest {
+    session_id: types::SessionId,
+    wallet_id: String,
+    data: String,
```

Comment:
Add comment explaining that the data is a string which will be hashed.

Reply:
Wasn't it clear enough? :rofl: 
I will include it. :+1:

Reply:
Ah, sorry, I meant a comment above the field:
```suggestion
    // Message to be signed
    data: String,
```